.travis.yml

```
@@ -1,24 +1,38 @@
 language: dart
 
 dart:
-  - 2.7.0
   - dev
 
-dart_task:
-  - test: --platform vm
-    xvfb: false
-  - test: --platform firefox
-  - dartanalyzer: --fatal-infos --fatal-warnings .
-
-matrix:
+jobs:
   include:
-  # Only validate formatting using the dev release
-  - dart: dev
-    dart_task: dartfmt
+    - stage: analyze_and_format
+      name: "Analyze lib/"
+      os: linux
+      script: dartanalyzer --enable-experiment=non-nullable --fatal-warnings --fatal-infos lib/
```

This bug should be fixed now, I don't think you should need the flag for the `dartanalyzer` call. You still need it for `pub run test` due to https://github.com/dart-lang/sdk/issues/43243

```suggestion
      script: dartanalyzer --fatal-warnings --fatal-infos lib/
```

---

I guess not? https://travis-ci.org/github/dart-lang/http_parser/jobs/727860944#L241

---

oh, I apparently forgot that the allow list is a thing. Sorry!